<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Ian R. Jones </title> <meta name="author" content="Ian R. Jones"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://enrach21.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Ian</span> R. Jones </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/books/">bookshelf</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="longhurst_prc21_2025" class="col-sm-8"> <div class="title">The PRC2.1 subcomplex opposes G1 progression through regulation of CCND1 and CCND2</div> <div class="author"> Adam D Longhurst, Kyle Wang, Harsha Garadi Suresh, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Mythili Ketavarapu, Henry N Ward, Ian R Jones, Vivek Narayan, Frances V Hundley, Arshia Zernab Hassan, Charles Boone, Chad L Myers, Yin Shen, Vijay Ramani, Brenda J Andrews, David P Toczyski' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>eLife</em>, Feb 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.97577" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Progression through the G1 phase of the cell cycle is the most highly regulated step in cellular division. We employed a chemogenetic approach to discover novel cellular networks that regulate cell cycle progression. This approach uncovered functional clusters of genes that altered sensitivity of cells to inhibitors of the G1/S transition. Mutation of components of the Polycomb Repressor Complex 2 rescued proliferation inhibition caused by the CDK4/6 inhibitor palbociclib, but not to inhibitors of S phase or mitosis. In addition to its core catalytic subunits, mutation of the PRC2.1 accessory protein MTF2, but not the PRC2.2 protein JARID2, rendered cells resistant to palbociclib treatment. We found that PRC2.1 (MTF2), but not PRC2.2 (JARID2), was critical for promoting H3K27me3 deposition at CpG islands genome-wide and in promoters. This included the CpG islands in the promoter of the CDK4/6 cyclins CCND1 and CCND2, and loss of MTF2 lead to upregulation of both CCND1 and CCND2. Our results demonstrate a role for PRC2.1, but not PRC2.2, in antagonizing G1 progression in a diversity of cell linages, including chronic myeloid leukemia (CML), breast cancer, and immortalized cell lines.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="cui_comparative_2025" class="col-sm-8"> <div class="title">Comparative characterization of human accelerated regions in neurons</div> <div class="author"> Xiekui Cui, Han Yang, Charles Cai, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Cooper Beaman, Xiaoyu Yang, Hongjiang Liu, Xingjie Ren, Zachary Amador, Ian R. Jones, Kathleen C. Keough, Meng Zhang, Tyler Fair, Armen Abnousi, Shreya Mishra, Zhen Ye, Ming Hu, Alex A. Pollen, Katherine S. Pollard, Yin Shen' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Apr 2025 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41586-025-08622-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="liang_conserved_2024" class="col-sm-8"> <div class="title">A conserved molecular logic for neurogenesis to gliogenesis switch in the cerebral cortex</div> <div class="author"> Xiaoyi G. Liang, Kendy Hoang, Brandon L. Meyerink, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Pratiksha Kc, Kitt Paraiso, Li Wang, Ian R. Jones, Yue Zhang, Sol Katzman, Thomas S. Finn, Jeremiah Tsyporin, Fangyuan Qu, Zhaoxu Chen, Axel Visel, Arnold Kriegstein, Yin Shen, Louis-Jan Pilaz, Bin Chen' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, May 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.2321711121" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>During development, neural stem cells in the cerebral cortex, also known as radial glial cells (RGCs), generate excitatory neurons, followed by production of cortical macroglia and inhibitory neurons that migrate to the olfactory bulb (OB). Understanding the mechanisms for this lineage switch is fundamental for unraveling how proper numbers of diverse neuronal and glial cell types are controlled. We and others recently showed that Sonic Hedgehog (Shh) signaling promotes the cortical RGC lineage switch to generate cortical oligodendrocytes and OB interneurons. During this process, cortical RGCs generate intermediate progenitor cells that express critical gliogenesis genes Ascl1 , Egfr, and Olig2 . The increased Ascl1 expression and appearance of Egfr + and Olig2 + cortical progenitors are concurrent with the switch from excitatory neurogenesis to gliogenesis and OB interneuron neurogenesis in the cortex. While Shh signaling promotes Olig2 expression in the developing spinal cord, the exact mechanism for this transcriptional regulation is not known. Furthermore, the transcriptional regulation of Olig2 and Egfr has not been explored. Here, we show that in cortical progenitor cells, multiple regulatory programs, including Pax6 and Gli3, prevent precocious expression of Olig2 , a gene essential for production of cortical oligodendrocytes and astrocytes. We identify multiple enhancers that control Olig2 expression in cortical progenitors and show that the mechanisms for regulating Olig2 expression are conserved between the mouse and human. Our study reveals evolutionarily conserved regulatory logic controlling the lineage switch of cortical neural stem cells.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="yang_functional_2023" class="col-sm-8"> <div class="title">Functional characterization of Alzheimer’s disease genetic variants in microglia</div> <div class="author"> Xiaoyu Yang, Jia Wen, Han Yang, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Ian R. Jones, Xiaodong Zhu, Weifang Liu, Bingkun Li, Claire D. Clelland, Wenjie Luo, Man Ying Wong, Xingjie Ren, Xiekui Cui, Michael Song, Hongjiang Liu, Cady Chen, Nicolas Eng, Mirunalini Ravichandran, Yang Sun, David Lee, Eric Van Buren, Min-Zhi Jiang, Candace S. Y. Chan, Chun Jimmie Ye, Rushika M. Perera, Li Gan, Yun Li, Yin Shen' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Nature Genetics</em>, Oct 2023 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41588-023-01506-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="jones_high-throughput_2022" class="col-sm-8"> <div class="title">High-throughput CRISPRi and CRISPRa technologies in 3D genome regulation for neuropsychiatric diseases</div> <div class="author"> Ian R Jones, Xingjie Ren, and Yin Shen </div> <div class="periodical"> <em>Human Molecular Genetics</em>, Oct 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/hmg/ddac193" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract Advances in genomics have led to the identification of many risk loci with hundreds of genes and thousands of DNA variants associated with neuropsychiatric disorders. A significant barrier to understanding the genetic underpinnings of complex diseases is the lack of functional characterization of risk genes and variants in biological systems relevant to human health and connecting disease-associated variants to pathological phenotypes. Characterizing gene and DNA variant functions requires genetic perturbations followed by molecular and cellular assays of neurobiological phenotypes. However, generating null or mutant alleles is low throughput, making it impossible to characterize disease-associated variants in large quantities efficiently. CRISPR interference (CRISPRi) and CRISPR activation (CRISPRa) screens can be leveraged to dissect the biological consequences of the tested genes and variants in their native context. Nevertheless, testing non-coding variants associated with complex diseases remains non-trivial. In this review, we first discuss the current challenges of interpreting the function of the non-coding genome and approaches to prioritizing disease-associated variants in the context of the 3D epigenome. Second, we provide a brief overview of high-throughput CRISPRi and CRISPRa screening strategies applicable for characterizing non-coding sequences in appropriate biological systems. Lastly, we discuss the promising prospects of using CRISPR-based technologies to dissect DNA sequences associated with neuropsychiatric diseases.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ren_parallel_2021" class="col-sm-8"> <div class="title">Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath</div> <div class="author"> Xingjie Ren, Mengchi Wang, Bingkun Li, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'Kirsty Jamieson, Lina Zheng, Ian R. Jones, Bin Li, Maya Asami Takagi, Jerry Lee, Lenka Maliskova, Tsz Wai Tam, Miao Yu, Rong Hu, Lindsay Lee, Armen Abnousi, Gang Li, Yun Li, Ming Hu, Bing Ren, Wei Wang, Yin Shen' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>Science Advances</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/sciadv.abi4360" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A new CRISPR screening strategy allows characterizing enhancers for multiple genes associated with converging phenotypes. , Current pooled CRISPR screens for cis-regulatory elements (CREs), based on transcriptional output changes, are typically limited to characterizing CREs of only one gene. Here, we describe CRISPRpath, a scalable screening strategy for parallelly characterizing CREs of genes linked to the same biological pathway and converging phenotypes. We demonstrate the ability of CRISPRpath for simultaneously identifying functional enhancers of six genes in the 6-thioguanine–induced DNA mismatch repair pathway using both CRISPR interference (CRISPRi) and CRISPR nuclease (CRISPRn) approaches. Sixty percent of the identified enhancers are known promoters with distinct epigenomic features compared to other active promoters, including increased chromatin accessibility and interactivity. Furthermore, by imposing different levels of selection pressure, CRISPRpath can distinguish enhancers exerting strong impact on gene expression from those exerting weak impact. Our results offer a nuanced view of cis-regulation and demonstrate that CRISPRpath can be leveraged for understanding the complex gene regulatory program beyond transcriptional output at scale.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="rosen_hprep_2021" class="col-sm-8"> <div class="title">HPRep: Quantifying Reproducibility in HiChIP and PLAC-Seq Datasets</div> <div class="author"> Jonathan D. Rosen, Yuchen Yang, Armen Abnousi, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Jiawen Chen, Michael Song, Ian R. Jones, Yin Shen, Ming Hu, Yun Li' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Current Issues in Molecular Biology</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3390/cimb43020082" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>HiChIP and PLAC-Seq are emerging technologies for studying genome-wide long-range chromatin interactions mediated by the protein of interest, enabling more sensitive and cost-efficient interrogation of protein-centric chromatin conformation. However, due to the unbalanced read distribution introduced by protein immunoprecipitation, existing reproducibility measures developed for Hi-C data are not appropriate for the analysis of HiChIP and PLAC-Seq data. Here, we present HPRep, a stratified and weighted correlation metric derived from normalized contact counts, to quantify reproducibility in HiChIP and PLAC-Seq data. We applied HPRep to multiple real datasets and demonstrate that HPRep outperforms existing reproducibility measures developed for Hi-C data. Specifically, we applied HPRep to H3K4me3 PLAC-Seq data from mouse embryonic stem cells and mouse brain tissues as well as H3K27ac HiChIP data from human lymphoblastoid cell line GM12878 and leukemia cell line K562, showing that HPRep can more clearly separate among pseudo-replicates, real replicates, and non-replicates. Furthermore, in an H3K4me3 PLAC-Seq dataset consisting of 11 samples from four human brain cell types, HPRep demonstrated the expected clustering of data that could not be achieved by existing methods developed for Hi-C data, highlighting the need for a reproducibility metric tailored to HiChIP and PLAC-Seq data.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ypsilanti_transcriptional_2021" class="col-sm-8"> <div class="title">Transcriptional network orchestrating regional patterning of cortical progenitors</div> <div class="author"> Athéna R. Ypsilanti, Kartik Pattabiraman, Rinaldo Catta-Preta, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Olga Golonzhka, Susan Lindtner, Ke Tang, Ian R. Jones, Armen Abnousi, Ivan Juric, Ming Hu, Yin Shen, Diane E. Dickel, Axel Visel, Len A. Pennacchio, Michael Hawrylycz, Carol L. Thompson, Hongkui Zeng, Iros Barozzi, Alex S. Nord, John L. Rubenstein' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, Dec 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.2024795118" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Significance Development of cortical areas begins in cortical stem cells through the action of morphogens controlling the graded expression of transcription factors (TFs). Here, we have systematically identified the TFs and gene regulatory elements (REs) that together control regional pattering of the cortical progenitor zone; these data have led us to propose a cortical regionalization TF network. To identify REs active in this network, we performed TF chromatin immunoprecipitation followed by sequencing (ChIP-seq) and chromatin-looping conformation experiments as well as assays for epigenomic marks and DNA accessibility in purified ventricular zone (VZ) progenitor cells in wild-type and patterning mutant mice. This integrated approach has laid the foundations to identify a TF network and cortical VZ REs involved in cortical regional patterning. , We uncovered a transcription factor (TF) network that regulates cortical regional patterning in radial glial stem cells. Screening the expression of hundreds of TFs in the developing mouse cortex identified 38 TFs that are expressed in gradients in the ventricular zone (VZ). We tested whether their cortical expression was altered in mutant mice with known patterning defects ( Emx2, Nr2f1 , and Pax6 ), which enabled us to define a cortical regionalization TF network (CRTFN). To identify genomic programming underlying this network, we performed TF ChIP-seq and chromatin-looping conformation to identify enhancer–gene interactions. To map enhancers involved in regional patterning of cortical progenitors, we performed assays for epigenomic marks and DNA accessibility in VZ cells purified from wild-type and patterning mutant mice. This integrated approach has identified a CRTFN and VZ enhancers involved in cortical regional patterning in the mouse.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="song_cell-type-specific_2020" class="col-sm-8"> <div class="title">Cell-type-specific 3D epigenomes in the developing human cortex</div> <div class="author"> Michael Song, Mark-Phillip Pebworth, Xiaoyu Yang, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Armen Abnousi, Changxu Fan, Jia Wen, Jonathan D. Rosen, Mayank N. K. Choudhary, Xiekui Cui, Ian R. Jones, Seth Bergenholtz, Ugomma C. Eze, Ivan Juric, Bingkun Li, Lenka Maliskova, Jerry Lee, Weifang Liu, Alex A. Pollen, Yun Li, Ting Wang, Ming Hu, Arnold R. Kriegstein, Yin Shen' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Nov 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41586-020-2825-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhan_proximal_2019" class="col-sm-8"> <div class="title">Proximal recolonization by self-renewing microglia re-establishes microglial homeostasis in the adult mouse brain</div> <div class="author"> Lihong Zhan, Grietje Krabbe, Fei Du, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Ian Jones, Meredith C. Reichert, Maria Telpoukhovskaia, Lay Kodama, Chao Wang, Seo-hyun Cho, Faten Sayed, Yaqiao Li, David Le, Yungui Zhou, Yin Shen, Brian West, Li Gan' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>PLOS Biology</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1371/journal.pbio.3000134" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="song_mapping_2019" class="col-sm-8"> <div class="title">Mapping cis-regulatory chromatin contacts in neural cells links neuropsychiatric disorder risk variants to target genes</div> <div class="author"> Michael Song, Xiaoyu Yang, Xingjie Ren, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Lenka Maliskova, Bingkun Li, Ian R. Jones, Chao Wang, Fadi Jacob, Kenneth Wu, Michela Traglia, Tsz Wai Tam, Kirsty Jamieson, Si-Yao Lu, Guo-Li Ming, Yun Li, Jun Yao, Lauren A. Weiss, Jesse R. Dixon, Luke M. Judge, Bruce R. Conklin, Hongjun Song, Li Gan, Yin Shen' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>Nature Genetics</em>, Aug 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41588-019-0472-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Ian R. Jones. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>